R/paginate.R

```diff
@@ -58,10 +70,12 @@ req_paginate <- function(req,
 #'
 #' @inheritParams req_perform
 #' @param resp An HTTP [response].
+#' @param parsed The response parsed by the argument `parse_resp` of [req_paginate()].
 #' @param max_pages The maximum number of pages to request.
 #' @param progress Display a progress bar?
 #'
-#' @return A list of responses.
+#' @return A list of responses parsed with the `parse_resp` argument of
+#'   [req_paginate()].
```

Comment:
```suggestion
#'   [req_paginate()]. If this argument is not specified, it will be a list of responses.
```

Which I don't love because it makes the interface a bit type-unstable; but I don't see how to do better.